ClientApp/viking-vault/src/ExchangeForm.css

```diff
@@ -0,0 +1,143 @@
+.background-div {
+    position: absolute;
+    background-color: #B0B2B6;
+    height: 100%;
+    width: 100%;
+}
+
+.input-field {
+    width: 50%;
+    padding: 0;
+    right: 1%;
+    position: relative;
+}
+
+.total-balance {
+    font-size: 150%;
+    font-family: Helvetica;
+    font-weight: 750;
+    color:gray;
+}
+
+.left-column {
+    position: absolute;
+    top: 11%;
+    left: 14%;
+}
+
+.right-column {
+    position: absolute;
+    left: 55%;
+    top: 16%;
+}
+
+.total-balance-text {
+    z-index: 2;
+    position: absolute;
+    left: 50%;
+    top: 66%;
+    color: gray;
+}
+
+#amount-div { 
```

Comment:
Again in here, what if I decide to change the HTML element I use for displaying the `#amount` from a `div` to a `span`, then this class name wouldn't make sense anymore. By naming classes like this you create potential confusion and you create a sort of an informal coupling

Reply:
Done. Ty.